hvpy/datasources.py

```diff
@@ -0,0 +1,504 @@
+    """
+
+    XRT_BE_MED_OPEN = 54
+    """
+    Hinode instrument XRT Be_med/Open.
+    """
+
+    XRT_BE_MED_TI_POLY = 55
+    """
+    Hinode instrument XRT Be_med/Ti_poly.
+    """
+
+    XRT_BE_THIN_AL_MESH = 56
+    """
+    Hinode instrument XRT Be_thin/Al_mesh.
+    """
+
+    XRT_BE_THIN_AL_THICK = 57
+    """
+    Hinode instrument XRT Be_thin/Al_thick.
+    """
+
+    XRT_BE_THIN_BE_THICK = 58
+    """
+    Hinode instrument XRT Be_thin/Be_thick.
+    """
+
+    XRT_BE_THIN_GBAND = 59
+    """
+    Hinode instrument XRT Be_thin/GBand.
+    """
+
+    XRT_BE_THIN_OPEN = 60
+    """
+    Hinode instrument XRT Be_thin/Open.
+    """
+
+    XRT_BE_THIN_TI_POLY = 61
+    """
+    Hinode instrument XRT Be_thin/Ti_poly.
+    """
+
+    XRT_C_POLY_AL_MESH = 62
+    """
+    Hinode instrument XRT C_poly/Al_mesh.
+    """
+
+    XRT_C_POLY_AL_THICK = 63
+    """
+    Hinode instrument XRT C_poly/Al_thick.
+    """
+
+    XRT_C_POLY_BE_THICK = 64
+    """
+    Hinode instrument XRT C_poly/Be_thick.
+    """
+
+    XRT_C_POLY_GBAND = 65
+    """
+    Hinode instrument XRT C_poly/GBand.
+    """
+
+    XRT_C_POLY_OPEN = 66
+    """
+    Hinode instrument XRT C_poly/Open.
+    """
+
+    XRT_C_POLY_TI_POLY = 67
+    """
+    Hinode instrument XRT C_poly/Ti_poly.
+    """
+
+    XRT_MISPOSITIONED_MISPOSITIONED = 68
+    """
+    Hinode instrument XRT Mispositioned/Mispositioned.
+    """
+
+    XRT_OPEN_AL_MESH = 69
+    """
+    Hinode instrument XRT Open/Al_mesh.
+    """
+
+    XRT_OPEN_AL_THICK = 70
+    """
+    Hinode instrument XRT Open/Al_thick.
+    """
+
+    XRT_OPEN_BE_THICK = 71
+    """
+    Hinode instrument XRT Open/Be_thick.
+    """
+
+    XRT_OPEN_GBAND = 72
+    """
+    Hinode instrument XRT Open/GBand.
+    """
+
+    XRT_OPEN_OPEN = 73
+    """
+    Hinode instrument XRT Open/Open.
+    """
+
+    XRT_OPEN_TI_POLY = 74
+    """
+    Hinode instrument XRT Open/Ti_poly.
+    """
+
+    TRACE_171 = 75
+    """
+    TRACE measurement 171.
+    """
+
+    TRACE_195 = 76
+    """
+    TRACE measurement 195.
+    """
+
+    TRACE_284 = 77
+    """
+    TRACE measurement 284.
+    """
+
+    TRACE_1216 = 78
+    """
+    TRACE measurement 1216.
+    """
+
+    TRACE_1550 = 79
+    """
+    TRACE measurement 1550.
+    """
+
+    TRACE_1600 = 80
+    """
+    TRACE measurement 1600.
+    """
+
+    TRACE_1700 = 81
+    """
+    TRACE measurement 1700.
+    """
+
+    TRACE_WHITE_LIGHT = 82
+    """
+    TRACE measurement white-light.
+    """
+
+    COSMO_KCOR = 83
+    """
+    MLSO instrument COSMO, detector KCor, measurement 735.
+    """
+
+    EUI_FSI_174 = 84
+    """
+    Solar Orbiter instrument FSI, measurement 174.
+    """
+
+    EUI_FSI_304 = 85
+    """
+    Solar Orbiter instrument FSI, measurement 304.
+    """
+
+    EUI_HRI_174 = 86
+    """
+    Solar Orbiter instrument HRI, measurement 174.
+    """
+
+    EUI_HRI_1216 = 87
+    """
+    Solar Orbiter instrument HRI, measurement 1216.
+    """
+
```

Comment:
A few more to add that are much more recent

|    88 | GOES-R SUVI 94                  |
|    89 | GOES-R SUVI 131                 |
|    90 | GOES-R SUVI 171                 |
|    91 | GOES-R SUVI 195                 |
|    92 | GOES-R SUVI 284                 |
|    93 | GOES-R SUVI 304                 |